README.md

```diff
@@ -2,12 +2,38 @@
 
 [![Travis CI](https://travis-ci.com/wangkuiyi/gotorch.svg?branch=develop)](https://travis-ci.com/wangkuiyi/gotorch)
 
-A Go-idiomatic binding of PyTorch, to be called by [Go+](https://github.com/goplus/gop) programs.
+GoTorch is a Go-idiomatic PyTorch, including PyTorch modules and functionals
+rewritten in Go.
 
-## Build and Test
+## From GoTorch to Go+Torch
 
-Please refer to [CONTRIBUTING.md](CONTRIBUTING.md).
+GoTorch work with the following open-source communities to form Go+Torch.
 
-## Examples
+- the [Go+](https://github.com/goplus/gop) community,
+- the PyTorch community, and
```

Comment:
Add links to  PyTorch and  TensorFlow XLA?

Reply:
I think it is OK to leave it as is. After we have some official collaboration, we can add their logos to our homepage. :-)